src/protobufs/gg.proto

```diff
@@ -0,0 +1,21 @@
+syntax = 'proto2';
+
+package gg.protobuf;
+
+message Function {
+  optional string exe = 1;
```

Comment:
In protobuf3, they removed the optional and required notation and everything is optional. For protobuf2, it's better to keep everything optional, because changing a required field to optional is not backward-compatible (it's not really an issue for us right now, but well :grinning:)